doc/dev/tma.md

```diff
@@ -435,4 +433,48 @@ or something else?).
 We do have the flexibility to arbitrarily transform and parallelize the non-tile branch,
 and scheduling this is very similar to how we schedule other fusions.
 
+### Code walk-through
+
+Please go to [test_tutorial.cpp](../../tests/cpp/test_tutorial.cpp) and search `TEST_F(Tutorial, BasicTMA)`.
+
 ### Examples
+
+#### Example 2: broadcast kernel with discontiguous input
```

Comment:
Example 1 will be `TEST_F(Tutorial, PointwiseTMA)` in https://github.com/NVIDIA/Fuser/pull/2148, thanks @rdspring1 for creating it!